public-html/contactus.php

```diff
@@ -3,16 +3,33 @@
 use src\SessionCookie;
 
 $session = new SessionCookie;
```

Comment:
maybe you can remove these line too, because its not been used in this code, right?

Reply:
It is starting the session right? Which is needed in all the files.

Reply:
oooooooooooo RIGHT! My bad forgot about it!